openedx/features/edly/api/tests/test_serializers.py

```diff
@@ -3,15 +3,24 @@
 """
 import json
 
+from django.conf import settings
 from django.core.validators import URLValidator
 from django.core.exceptions import ValidationError
 from django.test import TestCase, RequestFactory
+from django.test.utils import override_settings
+from mock import patch
```

Comment:
mock is third party and should be imported after json. Or in the same place u can replace `mock` with `unittest.mock`

Reply:
Done. `unittest.mock` was giving an `ImportError` so moved `from mock import patch` up.